x-pack/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -99,3 +99,7 @@ export const TRANSACTION_URL = 'transaction.page.url';
 export const CLIENT_GEO = 'client.geo';
 export const USER_AGENT_DEVICE = 'user_agent.device.name';
 export const USER_AGENT_OS = 'user_agent.os.name';
+
+export const TRANS_TIME_TO_FIRST_BYTE =
```

Comment:
I think we can just spell out TRANSACTION_TIME_TO_FIRST_BYTE, or if that's too long, TIME_TO_FIRST_BYTE. Not a huge fan of shortening TRANSACTION to TRANS. It's a loaded word, and it looks off IMO.